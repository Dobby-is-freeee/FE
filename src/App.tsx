function App() {
  return <div>농구 프로젝트</div>;
}

export default App;
