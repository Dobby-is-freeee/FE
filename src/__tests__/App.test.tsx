import { render } from '@testing-library/react';
import App from 'src/App';

describe('<App/>', () => {
  test('컴포넌트 스냅샷을 찍자.', () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
