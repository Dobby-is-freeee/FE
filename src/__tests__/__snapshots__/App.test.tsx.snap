// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> 컴포넌트 스냅샷을 찍자. 1`] = `
<div>
  <div>
    농구 프로젝트
  </div>
</div>
`;
